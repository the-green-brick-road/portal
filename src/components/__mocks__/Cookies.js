/* ------------------------------------------------------
# THE GREEN BRICK ROAD
# -------------------------------------------------------
# Copyright (c) [2023] The Green Brick Road
# All rights reserved
# -------------------------------------------------------
# Image component mock
# -------------------------------------------------------
# Nad√®ge LEMPERIERE, @24 may 2023
# Latest revision: 24 may 2023
# ---------------------------------------------------- */

/* Material UI includes */
import { Box, Typography }             from '@mui/material';

function Cookies(props) {

    /* --------- Gather inputs --------- */
    const { color } = props;

    /* -------- Defining theme --------- */
    return (
        <Box style={{ minHeight:'30px',zIndex:1000, position:'absolute', backgroundColor:color }} >
            <Typography> Cookies </Typography>
        </Box>
    );

}

export default Cookies;
