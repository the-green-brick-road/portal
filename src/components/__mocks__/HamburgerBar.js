/* ------------------------------------------------------
# THE GREEN BRICK ROAD
# -------------------------------------------------------
# Copyright (c) [2023] The Green Brick Road
# All rights reserved
# -------------------------------------------------------
# HamburgerBar component mock
# -------------------------------------------------------
# Nad√®ge LEMPERIERE, @24 may 2023
# Latest revision: 24 may 2023
# ---------------------------------------------------- */

/* Material UI includes */
import { Box, Typography }   from '@mui/material';

function HamburgerBar(props) {

    /* --------- Gather inputs --------- */
    const { height, isNegative, theme={} }  = props
    /* const componentName = 'HamburgerBar'; */

    /* -------- Defining theme --------- */
    let color = theme.palette.primary.main;
    if (isNegative) { color = theme.palette.common.white; }

    return (
        <Box style={{height:height, backgroundColor:color, width: '100%'}}>
            <Typography> HamburgerBar </Typography>
        </Box>
    );

}

export default HamburgerBar;
