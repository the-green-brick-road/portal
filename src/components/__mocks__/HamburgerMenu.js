/* ------------------------------------------------------
# THE GREEN BRICK ROAD
# -------------------------------------------------------
# Copyright (c) [2023] The Green Brick Road
# All rights reserved
# -------------------------------------------------------
# HamburgerMenu component mock
# -------------------------------------------------------
# Nad√®ge LEMPERIERE, @24 may 2023
# Latest revision: 24 may 2023
# ---------------------------------------------------- */

/* Material UI includes */
import { Box, Typography }   from '@mui/material';

function HamburgerMenu(props) {

    /* --------- Gather inputs --------- */
    const { top , margin, isNegative, theme={} } = props;
    /* const componentName = 'HamburgerBar'; */

    /* -------- Defining theme --------- */
    let color = theme.palette.primary.main;
    if (isNegative) { color = theme.palette.common.white; }
    const menuWidthString = `calc( 100vw - 2 * ${margin})`;

    return (
        <Box style={{ minHeight:'30px', width:menuWidthString, top:top, zIndex:1000, position:'absolute', backgroundColor:color }} >
            <Typography> HamburgerMenu </Typography>
        </Box>
    );

}

export default HamburgerMenu;