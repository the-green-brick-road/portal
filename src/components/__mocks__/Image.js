/* ------------------------------------------------------
# THE GREEN BRICK ROAD
# -------------------------------------------------------
# Copyright (c) [2023] The Green Brick Road
# All rights reserved
# -------------------------------------------------------
# Image component mock
# -------------------------------------------------------
# Nad√®ge LEMPERIERE, @24 may 2023
# Latest revision: 24 may 2023
# ---------------------------------------------------- */

/* Material UI includes */
import { Box }             from '@mui/material';

function Image(props) {

    /* --------- Gather inputs --------- */
    const { style={} }    = props;

    const local_style = JSON.parse(JSON.stringify(style))

    if (!('backgroundColor' in local_style)) { local_style['backgroundColor'] = 'blue' }

    /* ----------- Define HTML --------- */
    /* eslint-disable padded-blocks */
    return (
        <Box style={local_style} />
    );
    /* eslint-enable padded-blocks */

}

export default Image;
