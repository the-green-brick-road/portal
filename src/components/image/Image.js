/* ------------------------------------------------------
# THE GREEN BRICK ROAD
# -------------------------------------------------------
# Copyright (c) [2023] The Green Brick Road
# All rights reserved
# -------------------------------------------------------
# Image component
# -------------------------------------------------------
# Nad√®ge LEMPERIERE, @22 may 2023
# Latest revision: 14 june 2023
# ---------------------------------------------------- */

/* React includes */
import { Profiler }              from 'react';

/* Portal includes */
import { useDesign, useLogging } from '../../providers';

function Image(props) {

    /* --------- Gather inputs --------- */
    const { name, style={} }    = props;
    const { images }            = useDesign();
    const { onRender }          = useLogging();
    const componentName = 'Image';

    /* ---- Select image if loaded ----- */
    let data = {}
    if (name in images) { data = images[name] }
    const label_picture = `picture-${name}`

    /* ----------- Define HTML --------- */
    /* eslint-disable padded-blocks */
    return (
        <Profiler id={componentName} onRender={onRender}>
            {(data !== {}) && (
                <picture style={style} data-testid={label_picture}>
                    { Object.entries(data).toReversed().map((item) => {

                        let result;
                        const localType = `image/${item[0]}`;
                        const localSizes = Object.entries(item[1]);
                        let localSrcset = '';
                        for (let i = 0; i < localSizes.length; i += 1) {
                            localSrcset = `${localSrcset} ${localSizes[i][1].img} ${localSizes[i][1].width} ,`;
                        }
                        if (item[0] !== 'default') {
                            result = (<source key={item[0]} srcSet={localSrcset} type={localType} />);
                        }
                        return result;
                    })}
                    <img src={data.default} style={style} alt={name}/>
                </picture>
            )}
        </Profiler>
    );
    /* eslint-enable padded-blocks */

}

export default Image;
