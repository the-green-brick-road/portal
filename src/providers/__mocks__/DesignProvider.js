/* ------------------------------------------------------
# THE GREEN BRICK ROAD
# -------------------------------------------------------
# Copyright (c) [2023] The Green Brick Road
# All rights reserved
# -------------------------------------------------------
# Design provider mock
# -------------------------------------------------------
# NadÃ¨ge LEMPERIERE, @09 may 2023
# Latest revision: 09 may 2023
# ---------------------------------------------------- */

/* React includes */
import { createContext }       from 'react';

/* Material UI includes */
import { CssBaseline }                                 from '@mui/material';
import { ThemeProvider, createTheme }                  from '@mui/material/styles';

const context = createContext(null);

/* ----- Define provider values ---- */
var state = {
    isWebpSupported: true,
    isSliding:       false,
    isDarkMode:      false,
    images:          {},
    screen:          '',
    sizes:           {},
    setIsSliding:    jest.fn((value) => { state.isSliding = value }),
};

function DesignProvider(props) {

    const { children, isWebpSupported, screen, isDarkMode = false, isSliding = false, sizes, images = [], theme = {} } = props;

    /* ------ Defining behaviour ------- */
    var local_theme = {}
    if (Object.keys(theme).length !== 0) { local_theme = createTheme(theme); }

    state.isWebpSupported = isWebpSupported;
    state.screen          = screen;
    state.images          = {}
    state.sizes           = sizes;
    state.isSliding       = isSliding;
    state.isDarkMode      = isDarkMode;
    for (const image of images)
    {

        const rawext = image["raw"].split('.').pop();
        const webext = image["web"].split('.').pop();

        state.images[image.name] = {}
        state.images[image.name][rawext] = {}
        state.images[image.name][webext] = {}

        state.images[image.name][rawext]['small']           = {}
        state.images[image.name][rawext]['small']['img']    = require(`../../assets/small/${image.raw}`)
        state.images[image.name][rawext]['small']['width']  = `${sizes['small-width']}w`
        state.images[image.name][rawext]['medium']          = {}
        state.images[image.name][rawext]['medium']['img']   = require(`../../assets/medium/${image.raw}`)
        state.images[image.name][rawext]['medium']['width'] = `${sizes['medium-width']}w`
        state.images[image.name][rawext]['large']           = {}
        state.images[image.name][rawext]['large']['img']    = require(`../../assets/${image.raw}`)
        state.images[image.name][rawext]['large']['width']  = `${sizes['large-width']}w`

        state.images[image.name][webext]['small']           = {}
        state.images[image.name][webext]['small']['img']    = require(`../../assets/small/${image.web}`)
        state.images[image.name][webext]['small']['width']  = `${sizes['small-width']}w`
        state.images[image.name][webext]['medium']          = {}
        state.images[image.name][webext]['medium']['img']   = require(`../../assets/medium/${image.web}`)
        state.images[image.name][webext]['medium']['width'] = `${sizes['medium-width']}w`
        state.images[image.name][webext]['large']           = {}
        state.images[image.name][webext]['large']['img']    = require(`../../assets/${image.web}`)
        state.images[image.name][webext]['large']['width']  = `${sizes['large-width']}w`

        state.images[image.name]['default']= require(`../../assets/${image.raw}`)

    }

    /* ----------- Define HTML --------- */
    return (
        <ThemeProvider theme={local_theme}>
            <CssBaseline />
            <context.Provider value={state}>
                {children}
            </context.Provider>
        </ThemeProvider>
    );

}

function useDesign() { return state; };

export { useDesign, DesignProvider };