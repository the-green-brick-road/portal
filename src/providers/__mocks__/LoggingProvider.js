/* ------------------------------------------------------
# THE GREEN BRICK ROAD
# -------------------------------------------------------
# Copyright (c) [2023] The Green Brick Road
# All rights reserved
# -------------------------------------------------------
# Logging provider mock
# -------------------------------------------------------
# NadÃ¨ge LEMPERIERE, @09 may 2023
# Latest revision: 09 may 2023
# ---------------------------------------------------- */

/* React includes */
import { createContext }       from 'react';

const context = createContext(null);

/* ----- Define provider values ---- */
var state = {
    isLoggingActivated : true,
    logText: jest.fn((component, level, topic, message) => {  console.log(`${component} - ${level} - ${topic} -${message}`) }),
};

function LoggingProvider(props) {

    const { children } = props;

    /* ----------- Define HTML --------- */
    return (
        <context.Provider value={state}>
            {children}
        </context.Provider>
    );

}

function useLogging() { return state; };

export { useLogging, LoggingProvider };