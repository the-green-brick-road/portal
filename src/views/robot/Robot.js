/* ------------------------------------------------------
# THE GREEN BRICK ROAD
# -------------------------------------------------------
# Copyright (c) [2023] The Green Brick Road
# All rights reserved
# -------------------------------------------------------
# Robot view component
# -------------------------------------------------------
# NadÃ¨ge LEMPERIERE, @07 june 2023
# Latest revision: 12 june 2023
# ---------------------------------------------------- */

/* React includes */
import { Profiler, useState }                                from 'react';

/* Material UI includes */
import { Container, Typography, Stack, Divider, Pagination } from '@mui/material';
import { default as RadioButtonChecked }                     from '@mui/icons-material/RadioButtonChecked';
import { useTheme }                                          from '@mui/material/styles';

/* Portal includes */
import { useDesign, useLogging }                             from '../../providers';

function Robot(props) {

    /* --------- Gather inputs --------- */
    const { data }                            = props
    const theme                               = useTheme();
    const { sizes }                           = useDesign();
    const { logText, onRender }               = useLogging();
    const [ hoverFeature, setHoverFeature ]   = useState({id:"-1"})
    const [ selectFeature, setSelectFeature ] = useState({id:"-1"})
    const [ selectView, setSelectView ]       = useState(0)
    const componentName         = 'Robot';

    /* View management functions */
    const handleViewChange = (event, value) => {

        logText(componentName, 'debug', 'workflow', ' handleViewChange --- BEGIN');
        setSelectView(value - 1)
        setHoverFeature({id:"-1"})
        setSelectFeature({id:"-1"})

    };

    /* Feature info display management functions */
    const handleFeatureHover = (event, value) => {

        logText(componentName, 'debug', 'workflow', ' handleFeatureHover --- BEGIN');

        let id = -1
        if (event.target.parentNode.getAttribute('id') !== null) { id = event.target.parentNode.getAttribute('id') }
        else if (event.target.getAttribute('id') !== null) { id = event.target.getAttribute('id')}

        if (value) {

            for( let i_feature = 0; i_feature < data.features.length; i_feature ++) {

                if(data.features[i_feature].id === id ) { setHoverFeature(data.features[i_feature]) }

            }

        }
        else { setHoverFeature({id:"-1"}) }

    };

    /* Feature selection management functions */
    const handleFeatureClick = (event) => {

        logText(componentName, 'debug', 'workflow', ' handleFeatureClick --- BEGIN');

        let id = -1
        if (event.target.parentNode.getAttribute('id') !== null) { id = event.target.parentNode.getAttribute('id') }
        else if (event.target.getAttribute('id') !== null) { id = event.target.getAttribute('id')}

        if(selectFeature.id === id) { setSelectFeature({id:"-1"})}
        else {

            for( let i_feature = 0; i_feature < data.features.length; i_feature ++) {

                if(data.features[i_feature].id === id ) { setSelectFeature(data.features[i_feature]) }

            }

        }

    };

    /* ----------- Define HTML --------- */
    return (
        <Profiler id={componentName} onRender={onRender}>
            <Container style={{ backgroundColor:'#eeeeee', width:'100%', height:sizes['menu-height']}}/>
            <Container style={{ backgroundColor:'#eeeeee', fontFamily:data.font, fontSize:'14px', paddingTop:'20px', paddingBottom:'20px' }}>
                <Typography style={{ textAlign:'center', color:theme.typography.h1.color, fontFamily:data.font, fontSize:theme.typography.h1.fontSize}} >{data.name}</Typography>
            </Container>
            <Container style={{ position:'relative', backgroundColor:'#eeeeee', fontFamily:data.font, fontSize:'14px', padding:0}}>
                <img loading="lazy" src={data.views[selectView].image} style={{ position:'relative', width:'100%', height:data['image-height'], objectFit: 'cover', objectPosition: data['image-position'] }} alt={data.name}/>
                { ('features' in data.views[selectView]) && (data.views[selectView].features.map((jtem, jndex) => { /* Loop on all features to display them*/

                    const x_position = `${jtem.x}%`
                    const y_position = `${jtem.y}%`
                    const width =  `calc(100% - ${jtem.x}%)`

                    return(
                        <Container key={jtem.id} style={{position:'absolute', width:width, padding:0, top:y_position, left:x_position}}>
                            {((hoverFeature.id === jtem.id) || (selectFeature.id === jtem.id)) && (
                                <Stack direction="row" alignItems="center">
                                    <RadioButtonChecked id={jtem.id} onClick={handleFeatureClick} onMouseOver={(evt) => handleFeatureHover(evt, true)} onMouseOut={(evt) => handleFeatureHover(evt, false)} style={{color:theme.palette.common.white}}/>
                                    <Divider style={{ color:theme.palette.common.white, borderColor:theme.palette.common.white, width:'30px', paddingTop:5 }}/>
                                    <Typography style={{paddingLeft:'5px', color:theme.palette.common.white}}> {data.features[jtem.id].name} </Typography>
                                </Stack>
                            )}
                            {(hoverFeature.id !== jtem.id) && (selectFeature.id !== jtem.id) && (<RadioButtonChecked id={jtem.id} onClick={handleFeatureClick} onMouseOver={(evt) => handleFeatureHover(evt, true)} onMouseOut={(evt) => handleFeatureHover(evt, false)} style={{color:theme.palette.common.white}}/>)}

                        </Container>
                    )

                }))}
            </Container>
            <Stack direction="row" justifyContent="center"> <Pagination size="small" count={data.views.length} onChange={handleViewChange}/></Stack>
            <Container>
                <Typography>{selectFeature.name}</Typography>
                <Typography>{selectFeature.description}</Typography>
                {('media' in selectFeature) && (
                    <video autoPlay muted loop height="300">
                        <source src={selectFeature.media} type={selectFeature['media-type']}/>
                        Your browser does not support the video tag.
                    </video>
                )}
            </Container>

        </Profiler>
    );

}

export default Robot;