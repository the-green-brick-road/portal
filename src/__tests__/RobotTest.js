/* ------------------------------------------------------
# THE GREEN BRICK ROAD
# -------------------------------------------------------
# Copyright (c) [2023] The Green Brick Road
# All rights reserved
# -------------------------------------------------------
# Home view component test suite
# -------------------------------------------------------
# NadÃ¨ge LEMPERIERE, @30 may 2023
# Latest revision: 30 may 2023
# ---------------------------------------------------- */

/* Test includes */
import { render, prettyDOM }       from '@testing-library/react'
import { expect, test}             from '@jest/globals';

/* Component under test */
import { default as Robot }         from '../views/robot/Robot';

/* Mocks includes */
/* eslint-disable jest/no-mocks-import */
import { default as MockImage }    from '../components/__mocks__/Image';
import { useDesign as mockUseDesign, DesignProvider as MockDesignProvider } from '../providers/__mocks__/DesignProvider';
import { useLogging as mockUseLogging, LoggingProvider as MockLoggingProvider } from '../providers/__mocks__/LoggingProvider';
jest.mock("../components", () => ({ Image: (props) => MockImage(props) }));
jest.mock('../providers', () => ({
    useDesign: (() => { return mockUseDesign(); }),
    useLogging: (() => { return mockUseLogging(); }),
}));
/* eslint-enable jest/no-mocks-import */


describe("Robot view" ,() => {

    test('Should display robot page', async () => {

        const sizes = {
            "small-width"          : 200,
            "medium-width"         : 400,
            "large-width"          : 800,
            "menu-height"          : 80,
            "margin"               : 10,
            "hamburger-height"     : 40,
        }


        const view = render(
            <div>
                <MockLoggingProvider>
                    <MockDesignProvider sizes={sizes}>
                        <Robot data={{'name':'robot', 'id' : '1', 'type' : 'attachment', 'season' : '1', 'views' : [{'image':'view1.png'}], 'features' : [] }}/>
                    </MockDesignProvider>
                </MockLoggingProvider>
            </div>
        );
        const tree = prettyDOM(view.baseElement, Number.POSITIVE_INFINITY, {filterNode: () => true, escapeString: false, highlight: false});
        expect(tree).toMatchSnapshot();

    })

})
