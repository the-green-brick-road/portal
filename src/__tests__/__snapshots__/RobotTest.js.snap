// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Robot view Should display robot page 1`] = `
"<body>
  <div>
    <div>
      <div
        class=\\"MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root\\"
        style=\\"background-color: rgb(238, 238, 238); width: 100%; height: 80px;\\"
      />
      <div
        class=\\"MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root\\"
        style=\\"background-color: rgb(238, 238, 238); font-size: 14px; padding-top: 20px; padding-bottom: 20px;\\"
      >
        <p
          class=\\"MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root\\"
          style=\\"text-align: center; font-size: 6rem;\\"
        >
          robot
        </p>
      </div>
      <div
        class=\\"MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root\\"
        style=\\"position: relative; background-color: rgb(238, 238, 238); font-size: 14px; padding: 0px;\\"
      >
        <img
          alt=\\"robot\\"
          loading=\\"lazy\\"
          src=\\"view1.png\\"
          style=\\"position: relative; width: 100%; object-fit: cover;\\"
        />
      </div>
      <div
        class=\\"MuiStack-root css-13f3otb-MuiStack-root\\"
      >
         
        <nav
          aria-label=\\"pagination navigation\\"
          class=\\"MuiPagination-root MuiPagination-text css-1oj2twp-MuiPagination-root\\"
        >
          <ul
            class=\\"MuiPagination-ul css-wjh20t-MuiPagination-ul\\"
          >
            <li>
              <button
                aria-label=\\"Go to previous page\\"
                class=\\"MuiButtonBase-root Mui-disabled MuiPaginationItem-root MuiPaginationItem-sizeSmall MuiPaginationItem-text MuiPaginationItem-circular Mui-disabled MuiPaginationItem-previousNext css-1ba2zw7-MuiButtonBase-root-MuiPaginationItem-root\\"
                disabled=\\"\\"
                tabindex=\\"-1\\"
                type=\\"button\\"
              >
                <svg
                  aria-hidden=\\"true\\"
                  class=\\"MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiPaginationItem-icon css-tnzq9a-MuiSvgIcon-root-MuiPaginationItem-icon\\"
                  data-testid=\\"NavigateBeforeIcon\\"
                  focusable=\\"false\\"
                  viewBox=\\"0 0 24 24\\"
                >
                  <path
                    d=\\"M15.41 7.41L14 6l-6 6 6 6 1.41-1.41L10.83 12z\\"
                  />
                </svg>
              </button>
            </li>
            <li>
              <button
                aria-current=\\"true\\"
                aria-label=\\"page 1\\"
                class=\\"MuiButtonBase-root MuiPaginationItem-root MuiPaginationItem-sizeSmall MuiPaginationItem-text MuiPaginationItem-circular Mui-selected MuiPaginationItem-page css-1ba2zw7-MuiButtonBase-root-MuiPaginationItem-root\\"
                tabindex=\\"0\\"
                type=\\"button\\"
              >
                1
                <span
                  class=\\"MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root\\"
                />
              </button>
            </li>
            <li>
              <button
                aria-label=\\"Go to next page\\"
                class=\\"MuiButtonBase-root Mui-disabled MuiPaginationItem-root MuiPaginationItem-sizeSmall MuiPaginationItem-text MuiPaginationItem-circular Mui-disabled MuiPaginationItem-previousNext css-1ba2zw7-MuiButtonBase-root-MuiPaginationItem-root\\"
                disabled=\\"\\"
                tabindex=\\"-1\\"
                type=\\"button\\"
              >
                <svg
                  aria-hidden=\\"true\\"
                  class=\\"MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiPaginationItem-icon css-tnzq9a-MuiSvgIcon-root-MuiPaginationItem-icon\\"
                  data-testid=\\"NavigateNextIcon\\"
                  focusable=\\"false\\"
                  viewBox=\\"0 0 24 24\\"
                >
                  <path
                    d=\\"M10 6L8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z\\"
                  />
                </svg>
              </button>
            </li>
          </ul>
        </nav>
      </div>
      <div
        class=\\"MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root\\"
      >
        <p
          class=\\"MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root\\"
        />
        <p
          class=\\"MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root\\"
        />
      </div>
    </div>
  </div>
</body>"
`;
